kubernetes/samples/scripts/create-soa-domain/create-database-pv-pvc/db-pv.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: soadb-pv
+spec:
+  storageClassName: db-storage-class
+  capacity:
+    storage: 10Gi
+  accessModes:
+    - ReadWriteMany
+  # Valid values are Retain, Delete or Recycle
+  persistentVolumeReclaimPolicy: Retain
+  hostPath:
+  # nfs:
+    # server: %SAMPLE_STORAGE_NFS_SERVER%
+    path: "/scratch/k8s_dir/DB"
```

Comment:
Hard coding to /scratch makes the sample less portable.   Can the sample be altered to rely on "./kubernetes/samples/scripts/create-weblogic-domain-pv-pvc" to create the pv/c?

Reply:
Deleted this directory create-database-pv-pvc